site/content/docs/main/resource-filtering.md

```diff
@@ -320,4 +322,21 @@ Velero supported conditions and format listed below:
 
 **Resource policies rules**
 - Velero already has lots of include or exclude filters. the resource policies are the final filters after others include or exclude filters in one backup processing workflow. So if use a defined similar filter like the opt-in approach to backup one pod volume but skip backup of the same pod volume in resource policies, as resource policies are the final filters that are applied, the volume will not be backed up.
-- If volume resource policies conflict with themselves the first matched policy will be respected when many policies are defined.
\ No newline at end of file
+- If volume resource policies conflict with themselves the first matched policy will be respected when many policies are defined.
+
+### Restic configuration
+
+#### Restic file excludes
+
+See also https://restic.readthedocs.io/en/latest/040_backup.html#excluding-files
+
+```yaml
+version: v1
+resticPolicies:
+  - resticConfig:
+      excludes:
+        - /git/gitea-repositories/mirrors/**
+        - /indexers/**
+```
+
+NOTE: Currently only the first resticConfig is used.
```

Comment:
It would be good to explicitly state that the exclusion rules apply to all PVs backed up by Restic. I also wonder if allowing exclusions at PV level makes sense but it is certainly fine to have this as well.